### PR TITLE
server sends back json

### PR DESCRIPTION
**Pull request by K-Karneboge**
image

### CODE REVIEW

File: express-get-json/index.js [hunk 1]
```diff
@@ -0,0 +1,37 @@
...
+    name: 'Forbes Lindsay',
+    course: 'JavaScript',
+    score: 92
+  }
+};
+
+const gradesArray = [];
+for (var o in grades) {
+  gradesArray.push(grades[o]);
+}
```
Comment (scbowler) [inline express-get-json/index.js]: This should be inside the middleware function, you want it to regenerate the array with every request so if the grades object changes the client will always receive the most current information.

**Review by scbowler — CHANGES_REQUESTED** [review]
Looking good, just one update needed. Let me know if you have any questions.

File: express-get-json/index.js (hunk 1 above)
Comment (K-Karneboge) [reply express-get-json/index.js]: image
moved.

**Comment by K-Karneboge** [comment]
image
image

**Review by scbowler — APPROVED** [review]
Great job implementing the feedback, looks good! 👍

**Pull request merged**